quad/formats.go

```diff
@@ -0,0 +1,64 @@
+package quad
```

Comment:
Small comment is already in place in quad.go. Are you asking about extending it?